qml/InfoToolTip.qml

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import Theme 1.0
+
+Item {
+    anchors.fill: parent
+    property string text: ""
+    ToolTip {
+        id: tooltip
+        text: parent.text
+        delay: 250
```

Comment:
```suggestion
        delay: 0
```
If there is a delay it is a possibility that the user does not realize the help exists

Reply:
I'd keep the delay as this is how it works in all applications